reactor-core/src/test/java/reactor/core/publisher/MonoReduceSeedTest.java

```diff
@@ -312,6 +312,21 @@ public void scanSubscriber() {
 		assertThat(test.scan(Scannable.Attr.TERMINATED)).isFalse();
 		test.onError(new IllegalStateException("boom"));
 		assertThat(test.scan(Scannable.Attr.TERMINATED)).isTrue();
+	}
+
+	@Test
+	public void scanSubscriber1() {
```

Comment:
avoid non-descriptive prefixes like `1` when possible. this is more like `scanSubscriberCancelled`

Reply:
 ☑️